python/lsst/qserv/tests/unittest/testIntegration.py

```diff
@@ -47,10 +47,14 @@
 # Exported definitions --
 # -----------------------
 
+_LOG = logging.getLogger(__name__)
+
 
 class TestIntegration(unittest.TestCase):
 
     runMulti = False
+    qservServer = ''
+    czar_list = []
```

Comment:
Would be nice to add comments or docstrings for these. I knew this code at some point but even for me it's hard to read it without some explaining.